### DISCUSSION TITLE
Atrocore for Windows

### DISCUSSION THREAD

**Discussion by amulatri** [atrocore/atrocore#299] [General]
Hello, 
Is  AtroCore available for Windows OS for application development purpose? If yes then please links for download & installation instructions.

**Comment by atrolex** [comment]
AtroCore is a LAMP software. You still can run it on Windows, if you want. For that you need to install and configure Apache for Windows, MySQL for Windows (MariaDB is not supported, so XAMPP will not fit) and PHP for Windows (>7.4). Alternatively you can use WSL functionality for Windows (https://docs.microsoft.com/en-us/windows/wsl/). We will not create and support manuals for Windows, because web application performance on Windows in general is much worse than on Linux, so we do not recommend to use Windows as OS for production purposes. If you definitely want to run it on Windows Server, install there Ubuntu as a virtual machine (eg with the help of Virtualbox).

**Reply by amulatri** [reply]
I intent to use AtroCore on Windows OS for development purpose only. Production hosting will be on Linux.
